winml/lib/Common/inc/errors.h

```diff
@@ -5,32 +5,32 @@
 
 #include "core/common/status.h"
 
-#define WINML_THROW_IF_NOT_OK(status)                                                                 \
```

Comment:
>WINML_THROW_IF_NOT_OK [](start = 8, length = 21)

is this whole file just whitespace changes?

Reply:
Yep. It looks visual studio did clang format for this file.

---
In reply to: [355752302](https://github.com/microsoft/onnxruntime/pull/2596#discussion_r355752302) [](ancestors = 355752302)